### PR TITLE
Add deep link URL hash support

### PR DESCRIPTION
**Pull request by ebidel**
R: @arthurevans @garlicnation @cdata @rictic

Should be merged with https://github.com/PolymerElements/iron-component-page/pull/71

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -25,13 +25,13 @@
     "marked-element": "PolymerElements/marked-element#^1.0.0",
     "paper-styles": "PolymerElements/paper-styles#^1.0.0",
     "paper-button": "PolymerElements/paper-button#^1.0.0",
-    "polymer": "Polymer/polymer#^1.0.0",
+    "polymer": "Polymer/polymer#^1.2.3",
```
Comment (blasten) [inline bower.json]: I was wondering why the patch number is needed here. Does it fix a bug?

File: bower.json [hunk 2]
```diff
@@ -25,13 +25,13 @@
...
     "prism-element": "PolymerElements/prism-element#^1.0.2"
   },
   "devDependencies": {
     "test-fixture": "PolymerElements/test-fixture#^1.0.0",
     "iron-component-page": "PolymerElements/iron-component-page#^1.0.0",
-    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0",
+    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.18",
```
Comment (blasten) [inline bower.json]: same comment

File: iron-doc-property.css [hunk 3]
```diff
@@ -23,6 +23,11 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
   display: none;
 }
 
+.deeplink {
+  color: currentcolor;
+  text-decoration: none;
+}
+
```
Comment (blasten) [inline iron-doc-property.css:30]: Maybe we could also add a `:hover` rule,  e.g. `text-decoration:underline` or a different font color.  wdyt?

**Comment by blasten** [comment]
LGTM + minor comments

File: bower.json (hunk 1 above)
Comment (ebidel) [inline bower.json]: Not needed. I've just gotten  in the habit of keeping deps up to date. I can remove.

Comment (ebidel) [inline bower.json]: Done

File: bower.json (hunk 2 above)
Comment (ebidel) [inline bower.json]: Done

File: iron-doc-property.css (hunk 3 above)
Comment (ebidel) [inline iron-doc-property.css:30]: Done

**Comment by blasten** [comment]
:+1:

**Pull request merged**